docs/api.md

```diff
@@ -149,3 +149,18 @@ hardware does not match a known profile, the value "unknown" is used.
   host or there is a problem with the host itself. Refer to the
   *errorMessage* field in the status section for more details about
   the error condition.
+
+## Triggering Provisioning
+
+Several conditions must be met in order to initiate provisioning.
+
+1. The host spec must have an image with a URL pointing to a valid
+   image file visible from within the cluster and from the host
+   receiving the image.
+2. The host provisioning image URL field must be empty, indicating
```

Comment:
Nit: I suggest rewording this to focus on the state of the world primarily, and the contents of the status as a reflection. Something like "The host must not have an image currently provisioned, as reflected by `status.provisioning.image`."

Reply:
Done